masq/src/commands/financials_command.rs

```diff
@@ -128,32 +1311,1305 @@ mod tests {
         assert_eq!(
             *transact_params,
             vec![(
-                UiFinancialsRequest {}.tmb(0),
+                UiFinancialsRequest {
+                    stats_required: false,
+                    top_records_opt: Some(TopRecordsConfig {
+                        count: 123,
+                        ordered_by: TopRecordsOrdering::Balance
+                    }),
+                    custom_queries_opt: None
+                }
+                .tmb(0),
                 STANDARD_COMMAND_TIMEOUT_MILLIS
             )]
         );
-        assert_eq!(
-            stdout_arc.lock().unwrap().get_string(),
-            "\
-                Financial status totals in Gwei\n\
-                \n\
-                Unpaid and pending payable:       116688\n\
-                Paid payable:                     55555\n\
-                Unpaid receivable:                221144\n\
-                Paid receivable:                  66555\n"
-        );
-        assert_eq!(stderr_arc.lock().unwrap().get_string(), String::new());
     }
 
     #[test]
-    fn financials_command_sad_path() {
+    fn financials_command_allows_shorthands_including_custom_query() {
         let transact_params_arc = Arc::new(Mutex::new(vec![]));
+        let irrelevant_response = UiFinancialsResponse {
+            stats_opt: None,
+            query_results_opt: None,
+        };
+        let args = array_of_borrows_to_vec(&[
+            "financials",
+            "-g",
```

Comment:
Well...it might be a bit confusing, I admit...but the idea here is that input is always in MASQ (that's required...I did that so that I lightened the user's work at supplying values, since one will not want to type numbers with likely more than 9 digits, so you I expect them usually write quite a short number of MASQ, however, if they want they still can write a number like 45.555000006 to specify Gwei digits too) but the output's look depends on the argument...it's either displayed as MASQs or as Gweis.